java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -2743,6 +2744,135 @@ public CompletableFuture<Map<GlideString, Object>[]> xinfoConsumers(
                 response -> castArray(handleArrayResponseBinary(response), Map.class));
     }
 
+    @Override
+    public CompletableFuture<Object[]> xautoclaim(
+            @NonNull String key,
+            @NonNull String group,
+            @NonNull String consumer,
+            long minIdleTime,
+            @NonNull String start) {
+        String[] args = new String[] {key, group, consumer, Long.toString(minIdleTime), start};
+        return commandManager.submitNewCommand(XAutoClaim, args, this::handleArrayResponse);
+    }
+
+    @Override
+    public CompletableFuture<Object[]> xautoclaim(
+            @NonNull GlideString key,
+            @NonNull GlideString group,
+            @NonNull GlideString consumer,
+            long minIdleTime,
+            @NonNull GlideString start) {
+        GlideString[] args =
+                new GlideString[] {key, group, consumer, gs(Long.toString(minIdleTime)), start};
+        return commandManager.submitNewCommand(XAutoClaim, args, this::handleArrayResponse);
+    }
+
+    @Override
+    public CompletableFuture<Object[]> xautoclaim(
+            @NonNull String key,
+            @NonNull String group,
+            @NonNull String consumer,
+            long minIdleTime,
+            @NonNull String start,
+            long count) {
+        String[] args =
+                new String[] {
+                    key, group, consumer, Long.toString(minIdleTime), start, "COUNT", Long.toString(count)
```

Comment:
Can you use a constant for `COUNT` please?

Reply:
Can we steal a static string definition from other Options class, like StreamReadOptions? Because xautoclaim don't have a options class of itsown, and I don't know where else can we centrailzie the static string definition.

Reply:
I think yes